request.go

```diff
@@ -21,6 +22,16 @@ func (r *Request) Get(url string) (*Response, error) {
 	return r.client.doRequest(req)
 }
 
+// Put performs a request using the PUT method
+func (r *Request) Put(url string, body io.Reader) (*Response, error) {
+	req, err := r.client.newRequest(r.context, url, http.MethodPut, nil)
```

Comment:
Please specify the body when creating the request:
```go
req, err := r.client.newRequest(r.context, url, http.MethodPut, body)
```

Reply:
I fix it in latest change:
https://github.com/RafaelYon/requestfy/blob/467e6861bde086d2f9d1eb7c002fb0547874bfc7/request.go#L34